### ISSUE TITLE
Add migration samples for `AdaptiveGridView` (i.e. samples showing similar layout with `ItemsRepeater`)

### ISSUE THREAD

**Issue by michael-hawker** [CommunityToolkit/Windows#131]

**Comment by JimmyRespawn** [comment]
> Has there been any progress on this? I am using `AdaptiveGridView `in my app and using `ItemsRepeater` isn't an option because I couldn't find built-in support for selection, clicks etc. Would be interested in how this can be replicated with an ItemsRepeater. Can't migrate to v8 until this is possible

Yes, exactly. ItemsRepeater seems missing handful features from (adpative)gridview. The only way to migrate is to write your own adaptive gridview. Which won't be that hard.

**Comment by Arlodotexe** [comment]
> Has there been any progress on this? I am using `AdaptiveGridView `in my app and using `ItemsRepeater` isn't an option because I couldn't find built-in support for selection, clicks etc. Would be interested in how this can be replicated with an ItemsRepeater. Can't migrate to v8 until this is possible

@tipa You should already have all the tools needed to do this.
- The ListView or GridView (or AdaptiveGridView) derive from and build on ItemsControl to add click, selection, etc. 
- ItemsRepeater is an equivalent to ItemsControl. 
- If you're looking to replace the behavior of controls that derive ItemsControl, you'd look for controls that derive from ItemsRepeater instead of ItemsControl.

It sounds like you're looking for ItemsView instead of ItemsRepeater. See https://learn.microsoft.com/en-us/windows/apps/design/controls/itemsview
